core/io/file_access.cpp

```diff
@@ -69,6 +69,22 @@ void FileAccess::_set_access_type(AccessType p_access) {
 	_access_type = p_access;
 }
 
+uint16_t FileAccess::_endian_swap_16(uint16_t p_value) const {
```

Comment:
We already have macros for this, see `core/typedefs.h`

Reply:
Should I do that in a seperate PR, add a second commit, or amend to this one?

Reply:
Since it relates to this I'd say this one